routing/mock/dht.go

```diff
@@ -1,37 +0,0 @@
-package mockrouting
```

Comment:
So... I'm pretty sure we don't use this (pending CI). However, this, out of all this dead code, may actually be useful.